cloudbuild.yaml

```diff
@@ -12,20 +12,23 @@ steps:
       - DOCKER_FLOW_VERSION=${_VERSION}
       - '-t'
       - ${_IMAGE_NAME}:${_IMAGE_VERSION}
+      - '-t'
+      - ${_IMAGE_NAME}:latest
       - .
     id: Build
   - name: gcr.io/cloud-builders/docker
     args:
       - push
-      - ${_IMAGE_NAME}:${_IMAGE_VERSION}
+      - '-a'
+      - ${_IMAGE_NAME}
     id: Push
   - name: gcr.io/cloud-builders/docker
     id: Build deploy image using cache from previous build
     entrypoint: bash
     args:
       - '-c'
       - |
-        docker build -t ${_DEPLOY_IMAGE_NAME}:${_IMAGE_VERSION} --cache-from ${_DEPLOY_IMAGE_NAME}:latest -<<EOF
+        docker build -t ${_DEPLOY_IMAGE_NAME}:latest --cache-from ${_DEPLOY_IMAGE_NAME}:latest -<<EOF
```

Comment:
deploy helper image is always the same, so single `:latest` tag for it is enough